findSurfaceCharge.py

```diff
@@ -0,0 +1,163 @@
+from __future__ import print_function
+from pymol import cmd
+
+
+def findSurfaceAtoms(selection="all",cutoff=2.5):
+    """
+    Adapted from Jason Vertrees https://pymolwiki.org/index.php/FindSurfaceResidues
+DESCRIPTION
+
+    Finds those atoms on the surface of a protein
+    that have at least 'cutoff' exposed A**2 surface area.
+
+USAGE
+
+    findSurfaceAtoms [ selection, [ cutoff ]]
+
+SEE ALSO
+
+    findSurfaceResidues
+    """
+    cutoff = float(cutoff)
+
+    tmpObj = cmd.get_unused_name("_tmp")
+    cmd.create(tmpObj, "(" + selection + ") and polymer", zoom=0)
+
+    cmd.set("dot_solvent", 1, tmpObj)
+    cmd.get_area(selection=tmpObj, load_b=1)
+
+    # threshold on what one considers an "exposed" atom (in A**2):
+    cmd.remove(tmpObj + " and b < " + str(cutoff))
+
+    selName = cmd.get_unused_name("exposed_atm_")
+
+    cmd.select(selName, "(" + selection + ") in " + tmpObj)
+
+    cmd.delete(tmpObj)
+
+    return selName
+
+
+def findSurfaceCharge(pH=7.0, folded=True, selection="all", cutoff=2.5):
```

Comment:
Might be helpful to also return the charge so that it's easier for the caller to obtain.